src/Client.ts

```diff
@@ -110,8 +110,7 @@ export type ContentOptions = {
   /**
    * Allow configuring codecs for additional content types
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  codecs: ContentCodec<any>[]
+  codecs: ContentCodec[]
```

Comment:
with the type updates, the `ContentCodec` generic type no longer needs an explicit value. the default is `any`.